internal/api/v1/applications.go

```diff
@@ -12,8 +14,13 @@ import (
 	"github.com/epinio/epinio/internal/organizations"
 	"github.com/epinio/epinio/internal/services"
 	"github.com/julienschmidt/httprouter"
+	"github.com/pkg/errors"
 )
 
+// APIError struct is meant to host an error as described here:
+// https://jsonapi.org/examples/#error-objects-basics
+type APIError map[string]([]map[string]string)
```

Comment:
I believe this could do with separate type names for the inner elements, for higher readability.

Reply:
And it will make it easier to add functionality for returning multiple errors in one response.
I.e. function to create a single error entry, assemble multiple entries, wrap the response around such.